README.md

```diff
@@ -36,12 +36,18 @@ ota_0,    app,  ota_0,   0x10000, 0x180000,
 ota_1,    app,  ota_1,   0x190000, 0x180000,
 ```
 
-And tell espflash to use it in `Cargo.toml`. Read more in the [cargo espflash documentation]:
+And tell espflash to use it in `.cargo/config.toml` similar to the following according to your target. Read more in the [cargo espflash documentation](https://github.com/esp-rs/espflash/blob/main/cargo-espflash/README.md#bootloader-and-partition-table).
 ```toml
-[package.metadata.espflash]
-partition_table = "partitions.csv"
+[build]
+target = "xtensa-esp32s3-espidf"
+
+[target.xtensa-esp32s3-espidf]
+linker = "ldproxy"
+runner = "espflash flash --monitor --partition-table=partitions.csv"
```

Comment:
Do you have any reference to point to as to why this is the preferred method? I see that my previous method of adding it to the `Cargo.toml` metadata disappeared from the espflash docs as of the `3.0.0` release. But now they recommend setting the bootloader in `espflash.toml` (not `.cargo/config.toml`)